src/nsidc/metgen/metgen.py

```diff
@@ -73,6 +79,7 @@ def init_config(configuration_file):
     cfg_parser.set(DESTINATION_SECTION_NAME, "local_output_dir", Prompt.ask("Local output directory", default="output"))
     cfg_parser.set(DESTINATION_SECTION_NAME, "ummg_dir", Prompt.ask("Local UMM-G output directory (relative to local output directory)", default="ummg"))
     cfg_parser.set(DESTINATION_SECTION_NAME, "kinesis_stream_name", Prompt.ask("Kinesis stream name"))
+    cfg_parser.set(DESTINATION_SECTION_NAME, "staging_bucket_name", Prompt.ask("Cumulus s3 bucket name"))
```

Comment:
The value previously used in the template in the function s3_url (`s3://nsidc-cumulus-${environment}-ingest-staging/external/${auth_id}/${version}/${uuid}/${s3_name}`) is the "official" s3 url pattern. It makes sense to have a means of overriding the default, but my understanding is `s3://nsidc-cumulus-${environment}-ingest-staging/external/` should be the default value for the root of the staging bucket name.

Reply:
Based on the documentation for CNM messages, we'd want to flag any CNM content that didn't include the default pattern `s3://nsidc-cumulus-${environment}-ingest-staging/external/${auth_id}/${version}/${uuid}/${s3_name}` as invalid. We want to be able to test using non-official staging buckets, though, so this is more food for thought regarding the meaning of "valid" when it comes to CNM messages.

Reply:
I've reworked the default values, added tests, and set the default bucket & kinesis stream names to the cumulus patterns.